ctt-frontend/src/app/services/aws-category.service.spec.ts

```diff
@@ -1,16 +1,217 @@
-// import { TestBed } from '@angular/core/testing';
+import { TestBed } from '@angular/core/testing';
+import * as TypeMoq from 'typemoq';
+import { APIService } from './API.service';
 
-// import { AwsCategoryService } from './aws-category.service';
+import { AwsCategoryService } from './aws-category.service';
+import {
+  CreateCategoryInput as APICreateInput,
+  UpdateCategoryMutation,
+} from './API.service';
+import * as DummyData from './test-data/aws-category-service-data';
 
-// describe('AwsCategoryService', () => {
-//   let service: AwsCategoryService;
+describe('AwsCategoryService', () => {
+  let service: AwsCategoryService;
 
-//   beforeEach(() => {
-//     TestBed.configureTestingModule({});
-//     service = TestBed.inject(AwsCategoryService);
-//   });
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
```

Comment:
You can remove the `TestBed` setup from `beforeEach` and the `it should be created` test. Just create the `apiMock` and `sut` objects for each test in the `beforeEach` method manually and store them as global variables (like `service`).
Then you only have the `apiMock.configure(...)`  call in each test and you lose some lines in the test code.